website/blog/2021-02-22-7.13.0.md

```diff
@@ -0,0 +1,255 @@
+
+We welcome donations both from individual and from companies: if your company is interested in becoming a Gold tier sponsor ($1k/month) and wants to discuss more please reach out to [team@babeljs.io](mailto:team@babeljs.io)!
+
+## Highlights
+
+### Top-level `targets` option ([#12189](https://github.com/babel/babel/pull/12189), [RFC](https://github.com/babel/rfcs/pull/2))
+
+`targets` is the option that made `@babel/preset-env` the most powerful preset we have ever had: it can automatically choose what syntax to transform and what polyfills to inject based on your target browsers.
+
+However, other plugins can also benefit from knowing your targets. For example, you currently have to specify the targets twice if you are using our [new polyfill plugins](https://github.com/babel/babel-polyfills). With the new options you can lift it up to the top-level, specifying it only once:
+
+<table>
+<tr>
+<th>Old configuration</th>
+<th>New configuration</th>
+</tr>
+<tr>
+<td>
+
+```json
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": ">1%, not ie 11"
+    }]
+  ],
+  "plugins": [
+    ["polyfill-es-shims", {
+      "targets": ">1%, not ie 11"
+    }]
+  ]
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "targets": ">1%, not ie 11",
+  "presets": ["@babel/preset-env"],
+  "plugins": ["polyfill-es-shims"]
+}
+
+
+
+
+
+
+
+```
+
+</td>
+</tr>
+</table>
+
+We recommended converting configs to use the top-level `targets` option since:
+
+1. It better handles the [`esmodules: true`](TODO: Link) target. (it's intersected with the other targets, rather than replacing them)
+2. We are exploring how to use it in more plugins to generate more optimized output
+
+You can read the details about this new option in its [RFC](TODO: Link).
+
+> 🔮 In the future we might explore moving `@babel/preset-env` into `@babel/core` so that you don't have to install one additional package to get started with Babel. This new option can be seen as the first step in that direction!
+
+### Top-level `assumptions` option ([#12219](https://github.com/babel/babel/pull/12219), [RFC](https://github.com/babel/rfcs/pull/5))
+
+Many of our plugins have a `loose` option, which enables the compiler to make some assumptions about the code you are writing. It uses these to ignore certain edge cases and generate a smaller or faster output.
+
+However, `loose` has some problems: the term itself is not really descriptive of how it affects the compiler behaivior and sometimes must be set the same across different plugins to make sure that everything is compiled correctly.
+
+For this reason, we are introducing a new set of `assumptions` that you can set independently from the plugins you are using: a plugin can be influcenced by multiple assumptions and an assumption can influence multiple plugins.
+
+> :warning: This is an advanced functionality. Carefully choose which assumptions you enable, because they are not spec-compliant and might break your code if violated.
```

Comment:
```suggestion
> :warning: This is **advanced** functionality. Please be careful when enabling assumptions, because they are _not_ spec-compliant and may break your code in unexpected ways.
```